packages/vats/src/localchain.js

```diff
@@ -0,0 +1,89 @@
+import { E } from '@endo/far';
+import { M } from '@endo/patterns';
+
+const { Fail } = assert;
+
+export const LocalChainAccountI = M.interface('LocalChainAccount', {
+  getAddress: M.callWhen().returns(M.string()),
+  executeTx: M.callWhen(M.arrayOf(M.any())).returns(M.arrayOf(M.any())),
+  // A sample of adding specific queries.
+  allBalances: M.callWhen().returns(M.any()),
+});
+
+/** @param {import('@agoric/base-zone').Zone} zone */
+const prepareLocalChainAccount = zone =>
+  zone.exoClass(
+    'LocalChainAccount',
+    LocalChainAccountI,
+    (system, address, chain) => ({ system, address, chain }),
```

Comment:
please annotate signature so that the method returns aren't `any`

```suggestion
    /**
     * @param {import('./types.js').ScopedBridgeManager} system
     * @param {string} address
     * @param {{ query: (messages: any[]) => Promise<any> }} chain
     */
    (system, address, chain) => ({ system, address, chain }),
```